docs/docs/index.md

```diff
@@ -1,48 +1,39 @@
 # Welcome to RCJ Soccer Sim!
 
-The RoboCupJunior Soccer Simulator is an attempt at playing RoboCupJunior
-Soccer in a virtualized environment. It started as an experimental project organized by
-the RoboCupJunior Soccer Technical Committe in Feb 2021.
-
-![Screenshot of RCJ Soccer Sim](images/soccer_sim.png)
-
-The simulator is based on [Webots](https://github.com/cyberbotics/webots). The
-associated repository provides both the "automatic referee" (which implements
-the [Soccer Simulated Rules](https://github.com/RoboCupJuniorTC/soccer-rules-simulation))
+The RCJ Soccer Simulation League will be an additional league held at the 2021 RoboCup Junior.
+After a successful Demo competition in February, the next events will be
```

Comment:
```suggestion
After a successful Demo competition in February, the next event will be
```

The other option here would be to mention some other event (so that the plural makes sense)